R/plots.R

```diff
@@ -139,10 +140,11 @@ resetSearch <- function(pkgNamespaces) {
 #' @seealso \code{\link{addPlots}}, \code{\link{plotStudy}}
 #'
 #' @export
-getPlottingData <- function(study, modelID, featureID, libraries = NULL) {
+getPlottingData <- function(study, modelID, featureID, testID, libraries = NULL) {
```

Comment:
`testID` needs to be set to `NULL` as default since users may already have existing code that calls `getPlottingData()` with the argument `testID`.

```suggestion
getPlottingData <- function(study, modelID, featureID, testID = NULL, libraries = NULL) {
```